.gitlab-ci.yml

```diff
@@ -5,7 +5,7 @@ include:
 
 variables:
   CURRENT_CI_IMAGE: "13"
-  CI_IMAGE_DOCKER: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/dd-sdk-android:$CURRENT_CI_IMAGE
+  CI_IMAGE_DOCKER: registry.ddbuild.io/ci/dd-sdk-android:$CURRENT_CI_IMAGE
```

Comment:
By curiosity where do these address (`486234852809.dkr.ecr.us-east-1.amazonaws.com` and `registry.ddbuild.io`) come from, I don't find them from the link that you put in the description.